### ISSUE TITLE
[Bug]:  ConvTranspose2d is followed by BatchNorm2d and in_channels=out_channels=groups error,  Intel Xeon Gold-5318Y@2.10GHz   centos linux 7,  openvino 2023.1.0

### ISSUE THREAD

**Issue by xulingling516** [openvinotoolkit/openvino#22343]
### OpenVINO Version

2023.1.0

### Operating System

Other (Please specify in description)

### Device used for inference

CPU

### Framework

PyTorch

### Model used

mobilenet

### Issue description

ConvTranspose2d is followed by BatchNorm2d and in_channels=out_channels=groups, executing inference error on Intel Xeon Gold-5318Y@2.10GHz CPU: Segmentation fault (core dumped),  os: centos linux 7

### Step-by-step reproduction

[crash.zip](https://github.com/openvinotoolkit/openvino/files/14022326/crash.zip)

download the crash.zip,  uncompress it, python openvino_test1.py on Intel Xeon Gold-5318Y@2.10GHz,  centos linux 7,  openvino 2023.1.0,  error is : Segmentation fault (core dumped)

ConvTranspose2d is followed by BatchNorm2d and in_channels=out_channels=groups, executing inference error on Intel Xeon Gold-5318Y@2.10GHz CPU: Segmentation fault (core dumped),  os: centos linux 7

### Issue submission checklist

- [X] I'm reporting an issue. It's not a question.
- [X] I checked the problem with the documentation, FAQ, open issues, Stack Overflow, etc., and have not found a solution.
- [X] There is reproducer code and related data files such as images, videos, models, etc.

**Comment by mangguo321** [comment]
Thanks for reporting the issue. We reproduced it and debugging is WIP.

**Comment by mangguo321** [comment]
This issue is caused by a register conflict in jit_avx512_dw_f32 primitive in oneDNN. PR to fix: https://github.com/openvinotoolkit/oneDNN/pull/228

**Comment by yuxu42** [comment]
@xulingling516 could you please give a try with the fix PR to see whether it can solve the problem in your side? Thanks!

**Comment by xulingling516** [comment]
@yuxu42 ， I download openvino2024.0 and installed, but the problem was not solved. 
[crash2.zip](https://github.com/openvinotoolkit/openvino/files/14566474/crash2.zip)

**Comment by yuxu42** [comment]
@xulingling516 Could you please try out OpenVINO latest master or use this nightly https://pypi.org/project/openvino-nightly/ ?
